doc/wg/core/notes/core-notes-2025-01-10.md

```diff
@@ -0,0 +1,82 @@
+# Tock Meeting Notes 2025-01-10
+
+## Attendees
+
+- Johnathan Van Why
+- Amit Levy
+- Leon Schuermann
+- Tyler Potyondy
+- Branden Ghena
+- Alexandru Radovici
+- Benjamin Prevor
+- Brad Campbell
+- Hudson Ayers
+
+## Updates
+
+### Tock Release 2.2:
+- Leon: Tock 2.2 is released. 2 years between Tock 2.1 and Tock 2.2. Please let me know if there are any issues people find with this release.
+
+### Treadmill:
+- Leon: Treadmill is now generating reports for PRs. We now have a 3rd github action that reports success and does nothing else. This 3rd github action is used for testing. This is a bit hacky, but is mostly working.
+- Branden: Now Treadmill doesn't run until someone clicks merge?
+- Leon: It never ran before merge, we cannot have it run automatically. We have a planned change to test PRs when they are opened.
+- Branden: Is there a way for developers to manually run tests on branches?
+- Leon: This is only possible for branches that originate from the Tock repository.
+- Leon: Failures on PRs when merging are now reported and placed within the PR.
+- Leon: Please let Ben or myself know if anyone experiences issues or failures with Treadmill.
+
+### PSOC Board Support (https://github.com/tock/tock/pull/4300):
+- Alex: We added an infineon board. This is the PSOC. 
+- Leon: I saw this has a cortex-m0+ and a cortex-m4. Is there any reason you targeted this?
+- Alex: I will get back to you on this. I think it has something to do with how the code is uploaded.
+- Branden: I remember PSCOC having some hardware state machine similar to the RP2040. Are you going to try to support this?
+- Alex: No, not at this time. We selected this because we wanted to support an infineon chip and this was the chip infineon recommended us to work with. 
+
+### Meeting Logistics:
+- Amit: Moving forward we will switch these meetings to be Wednesdays, potentially moving the time 15-30 min later.
+- Amit: This change will start next week Wednesday. I will send an announcement.
```

Comment:
Should probably include that we moved the meeting to start at 9:15am Pacific Time (15 minutes later than the previous time) and shortened it to 45 minutes so it still ends at 10am.